### PR TITLE
Updated FAQ with correct link

### PR DESCRIPTION
**Pull request by nikkibeesetti**
“Will Spotify’s internal plugins be open sourced, too?” had an incorrect link, and I added the correct one.

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Review by stefanalund — CHANGES_REQUESTED** [review]
@nikkibeesetti this link is also incorrect. It misses a /backstage

**Review by nikkibeesetti — COMMENTED** [review]
reviewed

**Review by stefanalund — APPROVED** [review]
I fixed the links myself.

File: docs/FAQ.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ No, but it can be! Backstage is designed to be a developer portal for all your
 infrastructure tooling, services, and documentation. So, it's not a monitoring
 platform — but that doesn't mean you can't integrate a monitoring tool into
 Backstage by writing
-[a plugin](https://github.com/spotify/faq#what-is-a-plugin-in-backstage).
+[a plugin](https://github.com/spotify/backstage/faq#what-is-a-plugin-in-backstage).
```
Comment (fengypants) [inline docs/FAQ.md:18]: I don't think …/spotify/backstage/… works either. Think it has to be this:
https://github.com/spotify/backstage/blob/master/docs/FAQ.md#what-is-a-plugin-in-backstage

File: docs/FAQ.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ more, read our blog post,
 
 Yes, we've already started releasing open source versions of some of the plugins
 we use here, and we'll continue to do so.
-[Plugins](https://github.com/spotify/faq#what-is-a-plugin-in-backstage) are the
+[Plugins](https://github.com/spotify/backstage/faq#what-is-a-plugin-in-backstage) are the
```
Comment (fengypants) [inline docs/FAQ.md:39]: Same here — link should be:
https://github.com/spotify/backstage/blob/master/docs/FAQ.md#what-is-a-plugin-in-backstage

File: docs/FAQ.md [hunk 3]
```diff
@@ -129,15 +129,7 @@ can browse and search for all available plugins. ​
...
-​ By far, our most-used plugin is our TechDocs plugin, which we use for creating
-technical documentation. Our philosophy at Spotify is to treat "docs like code",
-where you write documentation using the same workflow as you write your code.
-This makes it easier to create, find, and update documentation. We hope to
-release
-[the open source version](https://github.com/spotify/backstage/issues/687) in
-the future. (See also:
-"[Will Spotify's internal plugins be open sourced, too?](https://github.com/spotify/faq#will-spotifys-internal-plugins-be-open-sourced-too)"
-above) ​
+By far, our most-used plugin is our TechDocs plugin, which we use for creating technical documentation. Our philosophy at Spotify is to treat "docs like code", where you write documentation using the same workflow as you write your code. This makes it easier to create, find, and update documentation. We hope to release [the open source version](https://github.com/spotify/backstage/issues/687) in the future. (See also: "[Will Spotify's internal plugins be open sourced, too?](https://github.com/spotify/faq#will-spotifys-internal-plugins-be-open-sourced-too)" above)
```
Comment (fengypants) [inline docs/FAQ.md:132]: This link should be:
https://github.com/spotify/backstage/blob/master/docs/FAQ.md#will-spotifys-internal-plugins-be-open-sourced-too

**Comment by Rugvip** [comment]
Or have the links be `/docs/FAQ.md#...`, it's shorter and will work better in editors and different branches.

**Comment by fengypants** [comment]
The shorter format works for me! Let's just make sure to update all **three** links with this PR.

**Pull request closed without merging**